contracts/cw1/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "cw1"
+version = "0.1.2"
+authors = ["Jan Woźniak <jan@confio.gmbh>"]
+edition = "2021"
+description = "todo"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/sylvia"
+homepage = "https://cosmwasm.com"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+cw-utils = "0.14.0" 
+cw1 = "0.14.0" 
+cw2 = "0.14.0" 
+cosmwasm-std = { version = "1.0.0", features = ["staking"] }
+cw-storage-plus = "0.14.0" 
+schemars = "0.8.1"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+thiserror = { version = "1.0.23" }
+sylvia = { version = "0.1.2", path = "../../sylvia" }
+sylvia-derive = {version = "0.1", path = "../../sylvia-derive" }
```

Comment:
I don't want any contracts to depend on `sylvia-derive` which is meant to be an only implementation detail. Everything is (or should be - if not, correct it) exposed via `sylvia` crate. If you have to include `sylvia-derive` in contracts/contract interfaces, it means it should be fixed in the framework.